packages/data/src/world/mm/snowhead_temple.yml

```diff
@@ -167,26 +167,34 @@
   dungeon: SH
   exits:
     "Snowhead Temple Compass Room": "has_explosives || trick_keg_explosives"
-    "Snowhead Temple Dual Switches": "small_keys_sh(3) || ((has_explosives || trick_keg_explosives) && small_keys_sh(2))"
+    "Snowhead Temple Dual Switches Locked": "small_keys_sh(3) || ((has_explosives || trick_keg_explosives) && small_keys_sh(2))"
   locations:
     "Snowhead Temple Icicle Room Alcove": "can_use_lens"
     "Snowhead Temple Icicle Room": "((has_arrows || has_mask_zora || can_use_lens) && can_break_boulders) || (can_hookshot_short && (has_explosives || trick_keg_explosives))"
     "Snowhead Temple Rupee 1": "has_arrows"
     "Snowhead Temple Rupee 2": "has_arrows"
     "Snowhead Temple Rupee 3": "has_arrows"
-"Snowhead Temple Dual Switches":
+"Snowhead Temple Dual Switches Locked":
   dungeon: SH
   exits:
     "Snowhead Temple Icicles": "small_keys_sh(3) || ((has_explosives || trick_keg_explosives) && small_keys_sh(2))"
-    "Snowhead Temple Center Level 2 Dual": "true"
+    "Snowhead Temple Dual Switches Unlocked": "has_mask_goron || has_mask_zora || short_hook_anywhere || has_hover_boots || can_use_fire_short_range || trick_sht_fireless_er"
```

Comment:
Could be wrong, but aren't there crates on the other side you can hookshot to?

Reply:
You're right. Does short hookshot reach the crates?

Reply:
It doesn't, but yeah long hookshot should be in logic for it.